test/selectors-relative.spec.js

```diff
@@ -1,33 +1,38 @@
-const Promise = require("promise");
-const express = require("express");
-const { firefox } = require("playwright");
+const fs = require("fs");
+const process = require("process");
+const { chromium } = require("playwright");
 const relativeSelector = require("../src/index");
-let server, port;
+
+const headless =
+  process.env.HEADFULL === undefined ||
+  process.env.HEADFULL.indexOf("true") < 0;
+
 let browser;
+let ctx;
```

Comment:
Consider renaming it to `context`

Reply:
No worry, first look tests that I have added.
Then I think I should discut about that: 
```
if (
    proximitySelector === "toLeftOf" ||
    proximitySelector === "toRightOf"
  ) {
    yDiff = yDiff * 10; // Give priority on distance X
  }
```
It to give a priority on the requested direction. When you find a composant at Left you expect that the components must be really at Left.
If we have a component at 20px left and top 0.
And a second at 2px left but 15px top.
That trick force to select the first one.

The value 10 look good but should be precise with real use case.

What do you think?